code/game/objects/items.dm

```diff
@@ -1052,6 +1052,10 @@ var/global/list/image/blood_overlays = list()
 /obj/item/proc/get_rating()
 	return 0
 
+// Like the above, but used for RPED sorting of parts.
+/obj/item/proc/rped_rating()
```

Comment:
Because ideally future things should use `get_rating` instead of reading the var directly, and I don't want things getting confused.